src/store/movie/operations/addMovieToNewListOperation.js

```diff
@@ -0,0 +1,45 @@
+import { createLogic } from 'redux-logic'
+import camelcaseKeys from 'camelcase-keys'
+
+import { openNotification } from 'Utils/'
+import { sessionIdSelector } from 'Store/auth/selectors'
+import { ADD_MOVIE_TO_NEW_LIST } from '../types'
+import {
+  addMovieToNewListStart,
+  addMovieToNewListSuccess,
+  addMovieToExistingList
+} from '../actions'
+
+export const addMovieToNewListOperation = createLogic({
+  type: ADD_MOVIE_TO_NEW_LIST,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { values, cb } = action
+    const { name, description } = values
+    const sessionId = sessionIdSelector(getState())
+    try {
+      dispatch(addMovieToNewListStart())
+      const newListResponse = await axios({
+        method: 'post',
+        url: '/list',
+        data: {
+          name,
+          description
+        },
+        params: {
+          session_id: sessionId
+        }
+      })
+      const { listId } = camelcaseKeys(newListResponse.data)
+      dispatch(addMovieToExistingList(listId))
+      cb()
```

Comment:
why do you need this callback?

Reply:
@VladWithCookies  yes, as we discussed for `createUserListOperation`

Reply:
do we really need to wait till operation complete to perform actions in callback?

Reply:
yes, I wanna reset the form and close the modal only in the success case